webpack/envs/clientCommonConfig.js

```diff
@@ -75,20 +77,14 @@ export const clientCommonConfig = {
       },
     }),
     // Remove moment.js localization files
-    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+    new webpack.IgnorePlugin({ resourceRegExp: /^\.\/locale$/ }),
+    new webpack.IgnorePlugin({ resourceRegExp: /moment$/ }),
     // Remove server-only modules from client bundles
-    ...[
-      // Remove server side of relay network layer.
-      new webpack.IgnorePlugin(
-        /^react-relay-network-modern-ssr\/node8\/server/
-      ),
-      // No matter what, we don't want the graphql-js package in client
-      // bundles. This /may/ lead to a broken build when e.g. a reaction
-      // module that's used on the client side imports something from
-      // graphql-js, but that's better than silently including this.
-      new webpack.IgnorePlugin(/^graphql(\/.*)?$/),
-    ],
-    new webpack.NamedModulesPlugin(),
+    // TODO: Why would these end up in the client bundle?
+    new webpack.IgnorePlugin({
+      resourceRegExp: /^react-relay-network-modern-ssr\/node8\/server/,
+    }),
```

Comment:
I think this was added due to how `Artsy/Router/buildServer|ClientApp` used to be exported from reaction, but we import these things directly everywhere now so i think this is safe to remove.